package.json

```diff
@@ -4,24 +4,38 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "webpack": "webpack",
+    "start": "gulp start"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "jest": "^20.0.4",
-    "webpack": "^3.0.0",
-    "webpack-dev-server": "^2.5.0",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-es2017": "^6.24.1",
+    "css-loader": "^0.28.4",
     "del": "^2.2.2",
-    "run-sequence": "^1.2.2"
+    "extract-text-webpack-plugin": "^2.1.2",
+    "jest": "^20.0.4",
+    "node-sass": "^4.5.3",
+    "postcss-loader": "^2.0.6",
+    "run-sequence": "^1.2.2",
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.18.2",
+    "webpack": "^3.1.0",
+    "webpack-dev-server": "^2.5.0"
   },
   "dependencies": {
     "chalk": "^2.0.1",
     "express": "^4.15.3",
     "gulp": "^3.9.1",
     "gulp-help": "^1.6.1",
     "gulp-nodemon": "^2.2.1",
+    "gulp-rename": "^1.2.2",
+    "lru-cache": "^4.1.1",
```

Comment:
I don't think we have direct dependency of `lrc-cache`, this shouldn't be added here